### PR TITLE
Adding Helm 3 support 

### PR DESCRIPTION
**Pull request by brendarearden**
Pulling in these changes https://github.com/lrills/helm-unittest/pull/89 
Tested against current master that is running Rancher Helm 2 and it appears to be backwards compatible

### CODE REVIEW

File: install-binary.sh [hunk 1]
```diff
@@ -5,15 +5,15 @@
 PROJECT_NAME="helm-unittest"
 PROJECT_GH="rancher/$PROJECT_NAME"
 
-: ${HELM_PLUGIN_PATH:="$(helm home)/plugins/helm-unittest"}
+#: ${HELM_PLUGIN_PATH:="$(helm home)/plugins/helm-unittest"}
```
Comment (dramich) [inline install-binary.sh]: These should just be deleted

**Comment by dramich** [comment]
Will this work with helm 3? I saw you tested with 2 and won't break 2.3 line.

**Comment by brendarearden** [comment]
It worked when tested against Helm v3.0.2

**Comment by brendarearden** [comment]
https://drone-pr.rancher.io/rancher/rancher/6748/1/2. build failure at helm-unittest install :(

**Comment by brendarearden** [comment]
It does work with helm 3 and appears to be backwards compatible so we should be good to merge. The only issue I am seeing is occasionally our build does not like the url

**Review by dramich — APPROVED** [review]

**Pull request merged**